app/views/coronavirus_form/check_your_answers/sections/_location.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  title: t("coronavirus_form.check_your_answers.sections.location.title"),
+  heading_level: 2,
+  heading_size: "l",
+  edit: {
+    href: "location?change-answer",
+  }
+} do %>
+  <dl class="govuk-summary-list">
```

Comment:
Agreed - it could be worth adding that feature to the component - but from just looking at this I don't think that this adds anything extra that isn't already in the summary list component. I'll have a dig and see.

Reply:
This is outdated 😺